src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperBlock.cs

```diff
@@ -42,6 +43,12 @@ public TagHelperBlock(TagHelperBlockBuilder source)
         }
 
         /// <summary>
+        /// Text used as a required prefix when matching HTML start and end tags in the Razor source to available 
+        /// tag helpers.
+        /// </summary>
+        public string Prefix { get; }
```

Comment:
this property should not exist.  it's currently used only when creating a `TagHelperBlockBuilder` from a `TagHelperBlock original`.  that overload is unused.